WordPress/src/test/java/org/wordpress/android/ui/mysite/MySiteViewModelTest.kt

```diff
@@ -1031,6 +1036,28 @@ class MySiteViewModelTest : BaseUnitTest() {
         verify(mySiteSourceManager).onQuickStartMenuInteraction(DynamicCardMenuInteraction.Remove(id))
     }
 
+    /* DASHBOARD TODAYS STATS CARD */
+
+    @Test
+    fun `given todays stat card, when card item is clicked, then stats page is opened`() =
+            test {
+                initSelectedSite()
+
+                requireNotNull(onTodaysStatsCardClick).invoke()
```

Comment:
`requireNotNull` call is redundant in the new test cases, can be removed.

Reply:
Yep 👍 . Done in 0740b82